stdlib/LinearAlgebra/src/qr.jl

```diff
@@ -366,17 +370,11 @@ true
     elementary reflectors, so that the `Q` and `R` matrices can be stored
     compactly rather as two separate dense matrices.
 """
-function qr(A::AbstractMatrix{T}, arg) where T
-    require_one_based_indexing(A)
-    AA = similar(A, _qreltype(T), size(A))
-    copyto!(AA, A)
-    return qr!(AA, arg)
-end
-function qr(A::AbstractMatrix{T}) where T
+function qr(A::AbstractMatrix{T}, arg...; kwargs...) where T
```

Comment:
`qr(A::AbstractMatrix{T}, arg)` and `qr(A::AbstractMatrix{T})` had almost identical function body. I think vararg is a cleaner solution to make it DRY. What do you think?